dizoo/board_games/go/envs/coords.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Google LLC
```

Comment:
can we overwrite these codes for reducing usage of other codes?

Reply:
we can do it later after the main part is done

Reply:
asap, to avoid some license problems